numba/core/decorators.py

```diff
@@ -11,6 +11,7 @@
 from numba.core.errors import DeprecationError, NumbaDeprecationWarning
 from numba.stencils.stencil import stencil
 from numba.core import config, sigutils, registry, cpu_dispatcher
+from numba.dppl import dppl_offload_dispatcher
```

Comment:
Let's remove this line. It is not needed

Reply:
It is needed for our target to be added to the registry